ceph.spec.in

```diff
@@ -1021,6 +1027,7 @@ if [ $FIRST_ARG -ge 1 ] ; then
   fi
   if [ "X$CEPH_AUTO_RESTART_ON_UPGRADE" = "Xyes" ] ; then
     /usr/bin/systemctl try-restart ceph-disk@\*.service > /dev/null 2>&1 || :
+    /usr/bin/systemctl try-restart ceph-volume@\*.service > /dev/null 2>&1 || :
```

Comment:
All the systemctl commands/macros support multiple arguments so you do not need to make a new call for ceph-volume*.